Libraries/Components/WebView/WebView.android.js

```diff
@@ -33,6 +34,15 @@ var WebViewState = keyMirror({
   ERROR: null,
 });
 
+var defaultRenderLoading = () => (
+  <View style={styles.loadingView}>
+    <ProgressBarAndroid
+      style={styles.loadingProgressBar}
+      styleAttr="Inverse"
```

Comment:
@andreicoman11 I wanted to be sure the spinner will be visible on white background since it's the default color.

> The "inverse" styles provide an inverse color scheme for the spinner, which may be necessary if your application uses a light colored theme (a white background).

[ProgressBar](http://developer.android.com/reference/android/widget/ProgressBar.html)

I just tried the `Normal` styleAttr, it seems to work well with the react-native theme. Should I remove it?